usaspending_api/agencies/management/commands/load_agencies_sql/toptier_agency_query.sql

```diff
@@ -0,0 +1,58 @@
+select
+    cgac_code,
```

Comment:
Would it be better to rename this single both cgac or frec values are populating this field?

Reply:
Troll much?

Just in case you aren't: https://federal-spending-transparency.atlassian.net/browse/DEV-3593

Reply:
It was an honest question. Thanks for the ticket.